### ISSUE TITLE
Import errors don't show up in junit report

### ISSUE THREAD

**Issue by sergeyKhristenko** [jest-community/jest-junit#212]
For example this test 
```
import stuff from 'notexistingmodule';

describe('Test Describe', function () {
  it('should works', function () {
    expect(true).toBeTruthy();
  });
});
```
fails in `jest` with
<img width="350" alt="image" src="https://user-images.githubusercontent.com/18219232/166431275-43c0b756-76cb-49cb-922e-8f8a457925be.png">
<img width="281" alt="image" src="https://user-images.githubusercontent.com/18219232/166431314-33e00588-bfd3-4bdf-a5ad-4e9a832b6afe.png">
But it doesn't show up in `junit` report at all:
(there were 2 test files launched - 1 passed and 1 with import error. The report shows only the passed one)
```
<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="jest tests" tests="1" failures="0" errors="0" time="3.858">
  <testsuite name="Test Component" errors="0" failures="0" skipped="0" timestamp="2022-05-03T09:24:30" time="2.897" tests="1">
    <testcase classname="jest/test-components-jest-test.js" name="Test Component::should support unit tests" time="0.018">
    </testcase>
  </testsuite>
</testsuites>
```

**Comment by SimenB** [comment]
Error is here: https://github.com/jest-community/jest-junit/blob/48a6c138aedfa9fed75500182d74e4649cc4a799/utils/buildJsonResults.js#L85-L88

`suite.testResults` is an empty array, but `suite.testExecError` is set to the import error

**Comment by palmerj3** [comment]
You need to enable the reportTestSuiteErrors option in order to have this functionality.

Reason being is that when jest cannot execute a test file reporters don't get very much info about it. So this is a separate option we provide.

**Comment by SimenB** [comment]
> Reason being is that when jest cannot execute a test file reporters don't get very much info about it. So this is a separate option we provide.

That seems unreasonable to me - the reporter is told what the error was. It should at least default to `true`. There _is_ an error to display, the reporter shouldn't obfuscate that (by default).

**Comment by palmerj3** [comment]
@SimenB the reason I've chosen this to not be on by default is that, when this happens, we cannot uniquely identify the suite(s).

We define the suite as an individual "describe" block.
So for errors, with this option enabled, it creates a placeholder entry in junit.xml.

So my reason for making this opt-in is that if people have systems to track tests and test suites over time this has the possibility of polluting that data given that it's a placeholder and we can't uniquely identify all suites/describes within a file jest can't parse.

**Comment by sergeyKhristenko** [comment]
> You need to enable the reportTestSuiteErrors option in order to have this functionality.
> 
> Reason being is that when jest cannot execute a test file reporters don't get very much info about it. So this is a separate option we provide.

That worked, thank you!
I did look on the docs and configs but apparently missed that option. I'd like to describe my case a little. We are using Jenkins and relying on `junit` plugin to set build status based on junit reports. So basically with this error the build is 🟢 and I noticed that accidentally looking through console logs. So I'd rather prefer `failed` build with not that precise error message than the false positive one. Or maybe some **bold** message in documentation describing this behaviour.
